app/less/admin.less

```diff
@@ -1,7 +1,12 @@
-#main-content {
+#main-content, #admin-main {
 	margin-top: 10%;
 }
 
+#admin-main a {
+  text-decoration: none;
+  color: #333333;

```

Comment:
I think, we need to create special file `variables.less` and extract all colors to this file and use it like `color: @color-admin-main-link`